samples/js/samples/classification.mjs

```diff
@@ -0,0 +1,16 @@
+import { printShape, printClass, getArrayByImgPath } from '../common/index.mjs';
+
+const modelPath = `../assets/models/v3-small_224_1.0_float.xml`;
+const imgPath = '../assets/images/coco224x224.jpg';
+const shape = [1, 224, 224, 3];
+const layout = 'NHWC';
+
+export default async function(openvinojs) {
+  const model = await openvinojs.loadModel(modelPath, shape, layout);
+  const inputTensor = await getArrayByImgPath(imgPath);
+  const shapeObj = new openvinojs.Shape(shape);
+  const outputTensor = await model.infer(inputTensor, shapeObj);
```

Comment:
According to common lib model.infer() first argument should be either ITensor object or a common js array numer[]. 
 This PR https://github.com/vishniakov-nikolai/openvino/pull/9 will solve this issue on node binding side and fix this example.
We should also make appropriate changes in the definitions of methods.

Reply:
Signature of this method will be refactored